pkg/store/kotsstore/downstream_store.go

```diff
@@ -470,6 +475,16 @@ func (s *KOTSStore) downstreamVersionFromRow(appID string, row scannable) (*down
 		}
 	}
 
+	if kotsAppSpecStr.String != "" {
+		obj, _, err := decode([]byte(kotsAppSpecStr.String), nil, nil)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to decode kots app spec yaml")
+		}
+		v.KotsAppApplication = obj.(*kotsv1beta1.Application)
```

Comment:
we already refer to this as `KotsApplication` in other places
```suggestion
		v.KotsApplication = obj.(*kotsv1beta1.Application)
```